webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -94,10 +94,14 @@ const coreHostsIndexColumns = [
   {
     columnName: 'last_report',
     title: __('Last report'),
-    wrapper: hostDetails => {
+    wrapper: ({ name, hostDetails }) => {
       const lastReport = hostDetails?.last_report;
-      return (
-        <RelativeDateTime defaultValue={__('Unknown')} date={lastReport} />
+      return lastReport ? (
+        <Link to={`hosts/${name}/config_reports/last`}>
```

Comment:
Rant: I would really love it if your API returned the `html_url` for objects, just like GitHub's API does. Then the front end doesn't need to construct them.

I wonder if we should instead link to the host by ID because that's more persistent (hosts can get renamed):
```suggestion
        <Link to={`hosts/${hostDetails.id}/config_reports/last`}>
```

But then I see others also link to a host by name, so I guess this is more consistent.

Reply:
yes, all the other links have the name.